src/shared/internal-calendar/internal-calendar.tsx

```diff
@@ -394,7 +407,27 @@ export const Component = (
         switch (type) {
             case "standalone":
                 return (
-                    <HeaderDropdown>{renderDropdownButtons()}</HeaderDropdown>
+                    <HeaderStandalone>
```

Comment:
the standalone header now seems identical to the input header? layout, text sizes and icon sizes look the same. can merge them instead